backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  description        String?
+  links              Json?
+  options            Json?
+  optOutText         String?                                    @map("opt_out_text")
+  hideFromListing    Boolean?                                   @map("hide_from_listing")
+  applicationSection MultiselectQuestionsApplicationSectionEnum @map("application_section")
+  jurisdictions      Jurisdictions[]
+  listings           ListingMultiselectQuestions[]
+
+  @@map("multiselect_questions")
+}
+
+model PaperApplications {
+  id                  String              @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt           DateTime            @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt           DateTime            @updatedAt @map("updated_at") @db.Timestamp(6)
+  language            LanguagesEnum
+  fileId              String?             @map("file_id") @db.Uuid
+  applicationMethodId String?             @map("application_method_id") @db.Uuid
+  assets              Assets?             @relation(fields: [fileId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicationMethods  ApplicationMethods? @relation(fields: [applicationMethodId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@map("paper_applications")
+}
+
+// Note: [name] formerly max length 256; [description] formerly max length 2048
+model ReservedCommunityTypes {
+  id             String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt      DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt      DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  name           String
+  description    String?
+  jurisdictionId String        @map("jurisdiction_id") @db.Uuid
+  listings       listings[]
+  jurisdictions  Jurisdictions @relation(fields: [jurisdictionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@map("reserved_community_types")
+}
+
+model Translations {
+  id             String         @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt      DateTime       @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt      DateTime       @updatedAt @map("updated_at") @db.Timestamp(6)
+  language       LanguagesEnum
+  translations   Json
+  jurisdictionId String?        @map("jurisdiction_id") @db.Uuid
+  jurisdictions  Jurisdictions? @relation(fields: [jurisdictionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@unique([jurisdictionId, language])
+  @@map("translations")
+}
+
+// Note: [name] formerly max length 256
+model UnitAccessibilityPriorityTypes {
+  id           String         @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt    DateTime       @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt    DateTime       @updatedAt @map("updated_at") @db.Timestamp(6)
+  name         String
+  units        Units[]
```

Comment:
Should these be relations?

Reply:
as mentioned above its the other side of a relationship so the work of relation things is done in the units model